autoendpoint/src/settings.rs

```diff
@@ -85,17 +87,30 @@ impl Settings {
         })
     }
 
-    /// Initialize the fernet encryption instance
-    pub fn make_fernet(&self) -> MultiFernet {
-        if !(self.crypto_keys.starts_with('[') && self.crypto_keys.ends_with(']')) {
-            panic!("Invalid AUTOEND_CRYPTO_KEY");
+    /// Convert a string like `[item1,item2]` into a iterator over `item1` and `item2`.
```

Comment:
technically, it's a JSON list, but I agree that pulling JSON into this is probably both overkill and a potential security issue.

Reply:
Unfortunately it's not a JSON list because the items aren't quoted.